src/components/organisms/UiMultipleChoices/UiMultipleChoices.vue

```diff
@@ -151,20 +163,15 @@ const setFirstMultipleAnswerItemRef = (
   const multipleChoicesItem = el as InstanceType<typeof UiMultipleChoicesItem>;
 
   if (multipleChoicesItem.content && multipleChoicesItem.content[0].content && elHasError) {
-    invalidMultipleChoicesItemRefs.set(idx, multipleChoicesItem.content[0].content.input);
+    invalidInputsRefs.set(idx, multipleChoicesItem.content[0].content.input);
```

Comment:
This still relinquishes the responsibility of editing `invalidInputsRefs` list to parent component, while this list should be kept up to date here with `invalidInputsRefs.delete()`

Reply:
To be sure, do you think about that:
```
if(elHasError) {
      if (multipleChoicesItem.content && multipleChoicesItem.content[0].content) {
        invalidInputsRefs.set(idx, multipleChoicesItem.content[0].content.input);
      }
    } else {
      invalidInputsRefs.clear()
    }
```
?

Reply:
No.